llvm/test/CodeGen/Generic/landingpad_wrong_type.ll

```diff
@@ -0,0 +1,18 @@
+; UNSUPPORTED: system-windows
```

Comment:
This doesn't require windows. Also, don't use generic codegen tests. Move to a target